src/Facebook/InstantArticles/Elements/Caption.php

```diff
@@ -344,4 +347,18 @@ public function toDOMElement($document = null)
 
         return $element;
     }
+
+    /**
+     * Overrides the @see TextContainer::isValid().
+     *
+     * @return true for valid Caption when it is filled, false otherwise.
+     */
+    public function isValid()
+    {
+        return
+            parent::isValid() ||
+            ($this->title && $this->title->isValid()) ||
+            ($this->subTitle && $this->subTitle->isValid()) ||
```

Comment:
Makes sense. Valdidate only if parent::isValid and $this->title.
Comming next push